FWCore/Framework/test/testOptions1_cfg.py

```diff
@@ -0,0 +1,10 @@
+# Test of options when all parameters taken from Config.py
+import FWCore.ParameterSet.Config as cms
+process = cms.Process("TEST")
+process.source = cms.Source("EmptySource")
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1)
+)
+process.options = dict (
+    dumpOptions = True
+)
```

Comment:
You should be able to do `process.options.dumpOptions = True`

Reply:
Done. It works. It is a little more concise. Thanks.